base/uk.ac.stfc.isis.ibex.opis/resources/stage/inout_motion_setpoints.opi

```diff
@@ -1,27 +1,27 @@
-<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
grep seems to suggest that this opi is not used anywhere, should we remove it?

Reply:
It's listed under opi_info as `Motion Set Point (Few)` did you mean that no one is using `It's listed under opi_info as `Motion Set Point (Few)` in a synoptic/device screen on any of the instruments?

Reply:
It's been a while but I think that is what I meant, it's probably fine to leave it though